lanelet2/lanelet2_core/src/LaneletMap.cpp

```diff
@@ -270,6 +300,27 @@ struct UsageLookup<Lanelet> {
       regElemLookup.insert(std::make_pair(elem, ll));
     }
   }
+  void remove(Lanelet ll, ConstLineString3d ls)
```

Comment:
When do we need to remove the linestring of a lanelet?

Reply:
oh that's right. we won't.